Documentation/faq.md

```diff
@@ -1,5 +1,11 @@
 ## Frequently Asked Questions (FAQ)
 
+### etcd, general
+
+#### Does the clients need to send requests to the leader?
```

Comment:
i think the clients need know all the server listening endpoints in order to send request right? so among the listening endpoints, which one will the client use? and how does the client know that sever is not down. if that server is down, then how will the client choose the next available one?